README.md

```
@@ -28,7 +28,7 @@ material icons.
 
 ## Table of Contents
 
-- [Key features](#key-features)
+- [Key features](#key-feature)
```

markdownlint-cli2 (0.17.2)

Link fragments should be valid
null

(MD051, link-fragments)